locales/en/data-classes.ftl

```diff
@@ -59,6 +59,7 @@ financial-investments = Financial investments
 financial-transactions = Financial transactions
 fitness-levels = Fitness levels
 flights-taken = Flights taken
+full-names = Full name
```

Comment:
@codemist 

I missed this one in review (not sure if it was there when I reviewed, but that's not relevant).

All categories in this file are plurals. Also, why is this category being added in this PR? Just a coincidence?

Reply:
Reusing strings is a bad idea, especially if we mix them from different sources. Some locales, for example, are translating categories lowercase.

Reply:
It's less than ideal in terms of localizability, but one way to minimize the impact would be to capitalize the first letter of the category (using a locale-sensitive library) when used in a different context.